### ISSUE TITLE
Indentation problems with multiple callbacks.

### ISSUE THREAD

**Issue by icirellik** [eslint/eslint#3615]
This is a stylistic question as much as a bug. We have a lot of code that looks like the following and since upgrading to the latest eslint are experiencing indentation errors that are unexpected. Is the formatting that eslint expects in this scenario the desired result. It seems strange to me. 

I've provided a simple reproducible example below using eslint 1.3.1.

```js
var Q = require('q');

function test() {
    return Q.fcall(function () {
        return true;
    }).then(function (result) {
        return result;
    }, function (err) {
        if (err) {
            throw new Error(err);
        }
    });
}

test();
```

```json
{
  "rules": {
    "indent": [2, 4]
  }
}
```

```sh
$ eslint --version
v1.3.1
$ eslint .

index.js
   9:9  error  Expected indentation of 12 space characters but found 8  indent
  12:5  error  Expected indentation of 8 space characters but found 4   indent

✖ 2 problems (2 errors, 0 warnings)

```

**Comment by ilyavolodin** [comment]
Exact duplicate of #3614 ?

**Comment by icirellik** [comment]
It appears so.

**Comment by ilyavolodin** [comment]
In that case closing in favor of #3614

**Comment by icirellik** [comment]
After reading the outcome of https://github.com/eslint/eslint/issues/3614, I would say this is a different issue as the first return here obviously isn't the end of the function chain.

**Comment by BYK** [comment]
@icirellik it is the same issue.
